src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java

```diff
@@ -51,15 +52,37 @@ public NSHMRuptureSetBuilder setPermutationStrategy(String permutationStrategyCl
 			super.setPermutationStrategy(RupturePermutationStrategy.valueOf(permutationStrategyClass));
 			return this;
 		}
+
+		/**
+		 * Sets the FaultModel file for all crustal faults
+		 * @param fsdFileName the XML FaultSection data file containing source fault information
+		 * @return this builder
+		 */
+		public CachedNSHMRuptureSetBuilder setFaultModelFile(String fsdFileName){
+			setFaultModelFile(new File(fsdFileName));
+			return this;
+		}
+
+		/**
+		 * Sets the subduction fault. At the moment, only one fault can be set.
+		 * @param faultName The name fo the fault.
+		 * @param fileName the CSV file containing all sections.
+		 * @return this builder
+		 */
+		public CachedNSHMRuptureSetBuilder setSubductionFault(String faultName, String fileName){
+			setSubductionFault(faultName, new File(fileName));
+			return this;
+		}
 		
 		/**
 		 * Caches the results of the build
 		 */
 		@Override
-		public SlipAlongRuptureModelRupSet buildRuptureSet(String fsdFileName) throws DocumentException, IOException {
-			ruptureSet = super.buildRuptureSet(fsdFileName);
+		public SlipAlongRuptureModelRupSet buildRuptureSet() throws DocumentException, IOException {
```

Comment:
@voj looks like the Gateway python clients will need a tweak for this change. I'll test for this and commit changes required before I approve - cheers

Reply:
👍

Reply:
No changes actually - my bad. All working as usual :+1: